client/src/components/BookingCards/BookingCard.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable prettier/prettier */
+import React  from "react";
+import useStyles from "./useStyles";
+import { 
+  Card, 
+  CardHeader, 
+  CardContent, 
+  IconButton, 
+  Typography, 
+  Link,
+  Box, 
+  Grid
+} from "@material-ui/core";
+import { Settings } from "@material-ui/icons";
+
+const mockImg = 'https://badairies.co.uk/assets/admin/plugins/images/users/4.jpg';
+
+interface Props {
+  createDate: any;
+  requests: any;
+  cardAction: boolean;
+  selectedBooking: string;
+  displayCardActions: any;
+  updateSelectedRequest: any;
+}
+
+interface Booking {
+  _id: string;
+  start: string;
+  end: string;
+  accepted: string;
+  declined: string;
+  user: { firstName: string; lastName: string };
+}
+
+function BookingCard({ 
+  createDate, 
+  requests, 
+  cardAction, 
+  selectedBooking, 
+  displayCardActions,
+  updateSelectedRequest}: Props ): JSX.Element {
+  
+  const classes = useStyles();
+
+  return (
+    <Box component="div" style={{height: 264, overflow: "auto"}}>
+      {requests.length 
+        ? requests.map((booking: Booking) => (
+          <Card key={booking._id} className={classes.cardBooking}>
+            <CardHeader 
+              classes={{title: classes.cardsBookingHeaderTitle}}
+              title={createDate(booking.start, booking.end)}
+              action={
+                <IconButton aria-label="settings" onClick={() => {
+                  displayCardActions(booking)}}>
+                  <Settings fontSize="small" color="disabled" />
+                  {(cardAction && selectedBooking === booking._id)
+                  ? (
+                    <Box className={classes.cardOptions}>
+                      <Typography>
+                        <Link href="#" onClick={(): void => {
+                          updateSelectedRequest(booking._id, "accepted")
+                          }}>Accept</Link>
+                      </Typography>
+                      <Typography>
+                        <Link href="#" onClick={(): void => {
+                          updateSelectedRequest(booking._id, "declined")
+                          }}>Decline</Link>
+                      </Typography>
+                    </Box>
+                  )
+                  : null
+                  }
+                </IconButton>
+              }
+              className={classes.cardBookingHeader} 
+            />
+            <CardContent className={classes.cardBookingContent}>
+              <Grid container>
+                <Grid item xs={8}>
+                  <Box className={classes.cardUserInfoBox} >
+                    <Box>
+                      <img src={mockImg} alt="user" className={classes.cardUserThumbnail}/>
```

Comment:
We should use profile pictures from the user and not mock images, we have support for this already available